.mailmap

```diff
@@ -114,3 +115,15 @@ Quinn Arbolante <quinn.arbolante@gmail.com> Cubostar <cubostar@gmail.com>
 Abhijeet Manhas <abhijeetmanhas720@gmail.com> abhijeetmanhas <abhijeetmanhas720@gmail.com>
 Sashank Mishra <sashankmishra27@gmail.com> sashank27 <sashankmishra27@gmail.com>
 Himanshu Pathak <hpathak336@gmail.com> himanshupathak21061998 <hpathak336@gmail.com>
+Ole Streicher <ole@aip.de> Ole Streicher <olebole@debian.org>
+Raahul Singh <raahulsingh002@gmail.com> Raahul Singh <raahul.s18@iiits.in>
+James Paul Mason <jmason86@gmail.com> James Mason <jmason86@gmail.com>
+Sarthak Jain <jains8844@gmail.com> Sarthak Jain <42547508+jains8844@users.noreply.github.com>
+Sarthak Jain <jains8844@gmail.com> jains8844 <jains8844@gmail.com>
+André Chicrala <andrechicrala@northumbria.ac.uk> Andre Chicrala <andrechicrala@northumbria.ac.uk>
+Joseph Letts <joe@fasteroute.com> Joe Letts <joe@fasteroute.com>
+James Calixto <jc2386@cornell.edu> jamescalixto <jc2386@cornell.edu>
+Trung Kien Dang <kiendang@users.noreply.github.com> Dang Trung Kien <kiendang@users.noreply.github.com>
+Swapnil Kannojia <swapnilkannojia123@gmail.com> swapnil99007 <swapnilkannojia123@gmail.com>
+Kris Akira Stern <krisastern@gmail.com> Kris Stern <krisastern@gobuddy.asia>
```

Comment:
I no longer use the email address `krisastern@gobuddy.asia`. Would it be possible to expunge it from the record?

Reply:
The `.mailmap` file is used so github knows that an old commit with the old email address should be attributed to the newer email address.

Reply:
Awesome! That's good to know.